src/test/java/duke/tasks/TaskManagerTest.java

```diff
@@ -0,0 +1,63 @@
+package duke.tasks;
```

Comment:
Just wanted to point out that the best-practice states that there's only one `assert*` per method (but I did it the same way as you too 🤣 )